packages/react-toolbar/src/components/ToolbarButton/ToolbarButton.types.ts

```diff
@@ -1,19 +1,17 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
-
-export type ToolbarButtonSlots = {
-  root: Slot<'div'>;
-};
-
-type ToolbarButtonCommons = {
-  // TODO Add things shared between props and state here
-};
+import type { ComponentProps, ComponentState } from '@fluentui/react-utilities';
+import { ButtonProps, ButtonSlots, ButtonState } from '@fluentui/react-button';
 
 /**
  * ToolbarButton Props
  */
-export type ToolbarButtonProps = ComponentProps<ToolbarButtonSlots> & ToolbarButtonCommons;
+export type ToolbarButtonProps = ComponentProps<ButtonSlots> &
+  Partial<Pick<ButtonProps, 'disabled' | 'disabledFocusable'>> & {
+    size?: 'small' | 'medium';
```

Comment:
is it a valid use case to have a `small` button in a `medium` toolbar ?

Reply:
Yeah, from design spec we gonna have it medium ( default ) and possibly small.